src/NuGet.Indexing/GalleryServiceImpl.cs

```diff
@@ -10,7 +10,7 @@ namespace NuGet.Indexing
 {
     public class GalleryServiceImpl
     {
-        public static void Search(JsonWriter jsonWriter, NuGetSearcherManager searcherManager, string q, bool countOnly, bool includePrerelease, string sortBy, int skip, int take, string feed, bool ignoreFilter, bool luceneQuery)
+        public static void Search(JsonWriter jsonWriter, NuGetSearcherManager searcherManager, string q, bool countOnly, bool includePrerelease, bool includeSemver2, string sortBy, int skip, int take, string feed, bool ignoreFilter, bool luceneQuery)
```

Comment:
Agree with @scottbommarito on the rename to `includeSemVer2`.
nit: align parameters vertically for readability and avoid horizontal scrolling (other places in this PR too).